book/push/push-android.md

```diff
@@ -0,0 +1,123 @@
+# Push plugin - Android
+
+The iOS push plugin for Zapp is based on implementing the `PushContract`.
+The protocol goes through all of the functions for initializing a push provider.
+
+## Create a new push provider
+
+This chapter describes how to build a push provider plugin. What kind of preparations are necessary, which interface methods should be implemented etc.
```

Comment:
This guide and then the `.` should be a `,`

Reply:
done